### PR TITLE
Docs: Add MCS doc

### LINKED ISSUE #587: Add a mcs example
**What would you like to be added**:

There is no related docs to describe how to use mcs. What about adding a mcs example? I wonder know how to use it, maybe somebody else has the same confusion as me.

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
Signed-off-by: changzhen <changzhen5@huawei.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Added a document for using the MCS feature.

**Which issue(s) this PR fixes**:
Fixes #587 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

cc @RainbowMango @Tingtal, can you help to check it out?

### CODE REVIEW

File: docs/mcs.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    - apiVersion: multicluster.x-k8s.io/v1alpha1
+      kind: ServiceImport
+      name: serve
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member2
+```
+
+### Step 3: Consume service on the `member2` cluster
```
Comment (RainbowMango) [inline docs/mcs.md:175]:
```suggestion
### Step 3: Consume service from `member2` cluster
```

File: docs/multi-cluster-service.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Multi-cluster Service Discovery
+
+Users are able to **export** and **import** services between clusters with [Multi-cluster Service APIs](https://github.com/kubernetes-sigs/mcs-api).
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install karmada refer to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/karmada-bootstrap.sh` script which is also used to run our E2E cases.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:9]: We can install Karmada by referring to...

File: docs/multi-cluster-service.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+### Member Cluster Network
+
+Ensure that at least two clusters has been added to karmada, and the container networks between member clusters are connected.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:13]: Ensure that at least two clusters have been...

File: docs/multi-cluster-service.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+- If you use the `hack/karmada-bootstrap.sh` script to deploy karmada. The deployed karmada has three member clusters, and the container networks of the `member1` and `member2` are connected.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:15]: - If you use the `hack/karmada-bootstrap.sh` script to deploy Karmada, Karmada will have three member clusters, and the container networks of the `member1` and `member2` will be connected.

File: docs/multi-cluster-service.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+- If you use `Kind` tool to create cluster and then join member cluster, you can refer to the [hack/util.sh](https://github.com/karmada-io/karmada/blob/af5f544cbe68d6b37a730b42dcc1ead0fac16915/hack/util.sh#L452-L470) script to enable the container network.
+
+### The ServiceExport and ServiceImport CRD has been installed
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:18]: ### The ServiceExport and ServiceImport CRDs have been installed

File: docs/multi-cluster-service.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+We need to install ServiceExport and ServiceImport in the member clusters.
+
+ServiceExport and ServiceImport have been installed on the **karmada control-plane**, we can create `ClusterPropagationPolicy` to propagate those two CRD to the member cluster.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:22]:
```suggestion
After ServiceExport and ServiceImport have been installed on the **karmada control-plane**, we can create `ClusterPropagationPolicy` to propagate those two CRDs to the member clusters.
```

File: docs/multi-cluster-service.md (hunk 2 above)
Comment (Tingtal) [inline docs/multi-cluster-service.md:9]:
```suggestion
We can install Karmada by referring to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/karmada-bootstrap.sh` script which is also used to run our E2E cases.
```

File: docs/multi-cluster-service.md (hunk 3 above)
Comment (Tingtal) [inline docs/multi-cluster-service.md:13]:
```suggestion
Ensure that at least two clusters have been added to Karmada, and the container networks between member clusters are connected.
```

File: docs/multi-cluster-service.md (hunk 4 above)
Comment (Tingtal) [inline docs/multi-cluster-service.md:15]:
```suggestion
- If you use the `hack/karmada-bootstrap.sh` script to deploy Karmada, Karmada will have three member clusters, and the container networks of the `member1` and `member2` will be connected.
```

File: docs/multi-cluster-service.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+### Karmada has been installed
+
+We can install karmada refer to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/karmada-bootstrap.sh` script which is also used to run our E2E cases.
+
+### Member Cluster Network
+
+Ensure that at least two clusters has been added to karmada, and the container networks between member clusters are connected.
+
+- If you use the `hack/karmada-bootstrap.sh` script to deploy karmada. The deployed karmada has three member clusters, and the container networks of the `member1` and `member2` are connected.
+- If you use `Kind` tool to create cluster and then join member cluster, you can refer to the [hack/util.sh](https://github.com/karmada-io/karmada/blob/af5f544cbe68d6b37a730b42dcc1ead0fac16915/hack/util.sh#L452-L470) script to enable the container network.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:16]:
```suggestion
- If you use `Kind` tool to create member clusters, you can refer to the [hack/util.sh](https://github.com/karmada-io/karmada/blob/af5f544cbe68d6b37a730b42dcc1ead0fac16915/hack/util.sh#L452-L470) script to enable the container networks.
```

File: docs/multi-cluster-service.md (hunk 5 above)
Comment (Tingtal) [inline docs/multi-cluster-service.md:18]:
```suggestion
### The ServiceExport and ServiceImport CRDs have been installed
```

File: docs/multi-cluster-service.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+```
+## Example
+
+### Step 1: Deploy service on the `member1` cluster 
+
+We need to deploy service on the `member1`cluster for discovery.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:61]:
```suggestion
We need to deploy service on the `member1` cluster for discovery.
```

File: docs/multi-cluster-service.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+      name: serve
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+### Step 2: Export service to the `member2` cluster
+
+- Create a `ServiceExport` object on **karmada control-plane**, and then create  a `PropagationPolicy` to propagate the `ServiceExport` object to the `member1` cluster.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:124]:
```suggestion
- Create a `ServiceExport` object on **karmada control-plane**, and then create a `PropagationPolicy` to propagate the `ServiceExport` object to the `member1` cluster.
```

File: docs/multi-cluster-service.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+    - apiVersion: multicluster.x-k8s.io/v1alpha1
+      kind: ServiceExport
+      name: serve
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+- Create a `ServiceImport` object on **karmada control-plane**, and then create a  `PropagationPlicy` to propagate the `ServiceImport` object to the `member2` cluster.
```
Comment (Tingtal) [inline docs/multi-cluster-service.md:147]:
```suggestion
- Create a `ServiceImport` object on **karmada control-plane**, and then create a `PropagationPlicy` to propagate the `ServiceImport` object to the `member2` cluster.
```

**Comment by XiShanYongYe-Chang** [comment]
@Tingtal Thank you very much for your review, and I've revised your suggestion.

**Pull request merged**